lib/gds_api/support_api.rb

```diff
@@ -47,4 +47,13 @@ def organisation(organisation_slug)
   def feedback_export_request(id)
     get_json!("#{endpoint}/anonymous-feedback/export-requests/#{id}")
   end
+
+  def problem_reports(options = {})
```

Comment:
can you document this please?
Example: https://github.com/alphagov/gds-api-adapters/pull/581/files#diff-c279de60f2b72b505bbabe7401804a88R98